Twig/Extension/RequireJSExtension.php

```diff
@@ -20,7 +20,7 @@
  * Twig extension providing RequireJS functionality.
  * @author Kevin Montag <kevin@hearsay.it>
  */
-class RequireJSExtension extends \Twig_Extension
+class RequireJSExtension extends \Twig_Extension implements \Twig_Extension_GlobalsInterface
```

Comment:
The `Twig_Extension_GlobalsInterface` has a method `getGlobals`, so by implementing the interface the class must have that method, but in this case the method is removed